nouns.json

```diff
@@ -113,7 +125,7 @@
     },
   },
   "0677022e": {
-    "root": {
+    "root": 
```

Comment:
I think you accidentally deleted a `{` here! It should be `"root": {`

Reply:
You are right.